content/software-triage.md

```diff
@@ -23,9 +23,9 @@ tableOfContents: true
 - [Live Disk](/articles/live-disk)
 - [Logs](/articles/log-files)
 
-## Suspend, Reboot, Shut Down
+## Shut Down
 
-Do the symptoms go away if the system is suspended, rebooted or shut down and then started up again? Sometimes repeated suspends or reboots are all that are needed to clear an issue out. It's also a good idea to test reboot and shutdown separately as they handle memory and power-management differently.
+Sometimes turning the machine off and on again is all that's needed to clear an issue out. Shut down your computer and power it back on to see if the issue persists.
```

Comment:
There can be markedly different behavior between shutdowns, reboots and suspends. I specified these out for a reason. :)